knotx-core/src/main/java/io/knotx/launcher/config/ConfProcessor.java

```diff
@@ -81,6 +90,33 @@ public void process(Vertx vertx, JsonObject configuration, Buffer input,
     );
   }
 
+  private class SysPropResolver implements ConfigResolver {
+
+    private final Properties sysProps = System.getProperties();
+
+    @Override
+    public ConfigValue lookup(String path) {
+      String value = sysProps.getProperty(path);
+      if (StringUtils.isNotBlank(value)) {
+        if (NumberUtils.isCreatable(value)) {
+          return ConfigValueFactory.fromAnyRef(NumberUtils.toInt(value));
+        }
+        if (Boolean.TRUE.toString().equalsIgnoreCase(value) || Boolean.FALSE.toString().equalsIgnoreCase("false")) {
```

Comment:
This statement will always return true because of `Boolean.FALSE.toString().equalsIgnoreCase("false")` ;)